camel/src/main/java/com/github/theprez/manzan/ManzanMessageFormatter.java

```diff
@@ -23,7 +23,7 @@ public static String format(final String _in, final Map<String, Object> _mapping
     private final String m_fmtStr;
 
     public ManzanMessageFormatter(final String _fmtStr) {
-        m_fmtStr = _fmtStr;
+        m_fmtStr = _fmtStr.toUpperCase();
```

Comment:
Will this change the casing of any text in their specified format? For example, for:
`format=$MESSAGE_ID$ (severity $SEVERITY$): $MESSAGE$`

the words "severity" would be in all caps?

Reply:
Fixed this